neo/test/iotest/test_nixio_fr.py

```diff
@@ -123,14 +124,16 @@ def test_annotations(self):
             bl.segments.append(seg)
             io.write_block(bl)
             io.close()
+
         with NixIOfr(filename=self.testfilename) as frio:
             frbl = frio.read_block()
             assert 'my_custom_annotation' in frbl.annotations
             assert 'something' in frbl.segments[0].annotations
-            # assert 'ansigrandom' in frbl.segments[0].analogsignals[0].annotations
+            assert 'ansigrandom' in frbl.segments[0].analogsignals[0].annotations
             assert 'railway' in frbl.segments[0].spiketrains[0].annotations
             assert 'venue' in frbl.segments[0].events[0].annotations
             assert 'evven' in frbl.segments[0].events[1].annotations
+            assert 'custom_id' in frbl.segments[0].analogsignals[0].array_annotations
```

Comment:
Could you also check for the size of of the value which must be channel count ?

Reply:
I thought that's part of the array_annotation tests. But I added a couple of lines to also cover this here.